sagemaker-pipelines/step-decorator/computer-vision-examples/computer-vision-pipeline.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>This dataset seems not to be hosted by AWS. Hoping to confirm if we are legally allowed to use it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I will send you the original SIM ticket with Legal where they approved the dataset. It was created by me to be published for a re:Invent workshop.

Reply:
Awesome! Thanks for the link. Looks good to me